mentorship ios/Views/Profile/ProfileEditor.swift

```diff
@@ -12,6 +12,13 @@ struct ProfileEditor: View {
     @State var editProfileData = ProfileViewModel().getEditProfileData()
     @ObservedObject var profileViewModel = ProfileViewModel()
     
+    var canSave:Bool {
+        guard let name = $editProfileData.name.wrappedValue, name.isEmpty == false else {
+            return false
+        }
+        return true
+    }
+    
```

Comment:
We won't need this code then.

Reply:
i made this way thinking about future proof, if later need to add another required field or another validation to enable/disable the button.

Reply:
Okay. In that case, we can define this computed property like this: 'return editProfileData.name.isEmpty'
Second, please add a unit test for this new computed property.

Reply:
done